frappe/erpnext
# feat: feature to create 1 PL for several SO irrespective of customers and t…

…o create several DNs grouped by each customer

Documentation Updates:

https://docs.erpnext.com/docs/v13/user/manual/en/stock/delivery-note/edit?wiki_page_patch=eb709e059f
https://docs.erpnext.com/docs/v13/user/manual/en/stock/pick-list/edit?wiki_page_patch=24f2742f11


**Test Cases:**

Customer is not mandatory in PL

Allow selection of multiple SO from several customers

Do not edit Picked Qty in PL. Submit the PL
	- Picked Qty should = Stock Qty

Let qty = 10. Enter 5 as picked qty and submit
	- Picked Qty should remain as 5 and should not be set to stock qty

On submit of PL
	-  “% Picked” (new field) should be updated in SO.
	- “Picked Qty”(new field) should be updated in corresponding SO item row

Create PL for an SO where 
	* “% Picked” is 100%. - On submission, should throw validation message
		* “% Picked” is < 100 % - Should allow submission until 100%

Create PL from multiple SO by clicking on ‘Get Items’
	- submit and create DN from PL 
	- should create separate DNs for each customer.

Create PL by adding items manually(not from SO). Then select an SO also. PL now has both items from SO and those not from SO
	- submit and create DN from PL
	- should create 1 DN for items added manually
	- separate DNs for items from SO, grouped by customer
